homeassistant/components/redgtech/quality_scale.yaml

```
@@ -0,0 +1,66 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: No custom actions
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: No custom actions
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: done
```

```suggestion
  entity-event-setup: exempt
```
comment: no explicit subscriptions to events